### ISSUE TITLE
How to finetune the model?

### ISSUE THREAD

**Issue by leiwen83** [visinf/irr#41]
Like if I try to do the quantization finetune over Sintel dataset, is there any parameter needed to be adjust, like learning rate, or something else?

Thx.

**Comment by minghuiwsw** [comment]
hi, @hurjunhwa , I noticed you provide the training script for the baseline: pwcnet. But I wonder if it is for flycharis_occ dataset only or it's applicable to other datasets. I mean, if I want to train pwcnet for other datasets to achieve the expected performance in paper, do I need to change the hypermeters in the config file of pwcnet.sh? If yes, can you provide me with those files? Thanks a lot!
